pulp_rpm/app/migrations/0044_modulemdobsolete.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.13 on 2022-06-17 14:08
+
+from django.db import migrations, models
```

Comment:
Since 6.11 is supposed to support EL9, and 6.11 is using pulpcore 3.16, we need to find a way to backport this functionality into a release that they can use.

Options are:

* Downstream upgrades pulpcore and pulp_rpm together
* We re-order the migrations again and backport this into 3.17, despite the migration
* We delay the features of 3.18 that required a pulpcore minimum version bump until a later release by pulling them out and putting them back

I'm leaning towards the 2nd option

Reply:
@ggainey @ipanova @pavelpicka Would like feedback from all of you on this.  It's a bit unfortunate that we're in this situation again, I think it's mostly my fault for overlooking this until after we merged RBAC

Reply:
I think the second bullet is the right choice (re-order migrations)

Reply:
Yup.  Let's have one commit for this change including the migration, with the appropriate name / number, and then a second commit to re-order the existing migrations to follow this one.